docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>"... internals of <code>aesara</code> RandomVariables and <code>pymc</code> distributions...”  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='114'/>

Reply:
+1